tools/smart_locator_finder.py

```
@@ -474,6 +474,360 @@ async def _find_element_by_description(page, description: str) -> Optional[str]:
+    
+    # Try each locator to find the cell
+    for cell_locator in locators_to_try:
+        try:
+            count = await page.locator(cell_locator).count()
+            
+            if count == 1:
+                # Cell found! Now determine what to return
+                
+                if expected_text:
+                    # Validate that expected_text is somewhere in this cell
+                    is_match, actual_text = await _validate_semantic_match(page, cell_locator, expected_text)
```

Ruff (0.14.8)

Unpacked variable `actual_text` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)